shared/src/types/transaction/protocol.rs

```diff
@@ -76,9 +76,8 @@ mod protocol_txs {
         DKG(DkgMessage),
         /// Tx requesting a new DKG session keypair
         NewDkgKeypair(Tx),
-        /// Aggregation of Ethereum state changes
-        /// voted on by validators in last block
-        EthereumStateUpdate(Tx),
+        /// Ethereum events contained in vote extensions
+        EthereumEventVoteExtensions,
```

Comment:
NB: this `ProtocolTxType` is guarded by the `ferveo-tpke` feature flag. I think we should consider removing that flag from this file altogether (or at least refactoring the module so that more generic protocol transaction types are not guarded by it).